api/python/t4/formats.py

```diff
@@ -134,15 +134,36 @@ def search(cls, obj_type=None, meta=None, ext=None):
             ext: The filename extension for the data
                 If given, then if other methods fail or are not specified,
                 the handler(s) for the extension `ext` will be returned.
+                If other methods do not fail, any handlers that support the
```

Comment:
Update the comment at line 125, it seems to conflict with this line

Reply:
done